WordPress/Docs/WP/DeprecatedParametersStandard.xml

```diff
@@ -0,0 +1,19 @@
+<documentation title="Deprecated Parameters">
+    <standard>
+    <![CDATA[
+    Please refrain from using deprecated WordPress Parameters. 
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: use of non deprecated parameters.">
```

Comment:
The negation in the current string is not quite right in English.

Change to: `Valid: not using deprecated parameters.`

Reply:
Changed this in all files in this commit.